database/mariadb/README.md

```diff
@@ -34,10 +34,10 @@ Bootstrap: docker
 From: mariadb:10.3.9
 
 %post
-# <YOUR_USERNAME> is the user who will be executing the container,
-# just run: `whoami` and that's your username.
-# eg. sed -ie "s/^#user.*/user = westleyk/" /etc/mysql/my.cnf
-sed -ie "s/^#user.*/user = <YOUR_USERNAME>/" /etc/mysql/my.cnf
+# replace `your-name` with your username, run `whoami` to see your username
+YOUR_USERNAME="your-name"
+
```

Comment:
Also export the username variable in here

Reply:
It's only used in the file ... there isn't a need to export it

Reply:
Yes, No, Maybe...

Reply:
`export`, or no `export`?